vllm/model_executor/layers/fused_moe/cutlass_moe.py

```diff
@@ -120,18 +134,22 @@ def cutlass_moe_fp8(
                                  dtype=torch.int32,
                                  device=device)
 
-    a_map = torch.empty((topk_ids.numel()), dtype=torch.int32, device=device)
-    c_map = torch.empty((topk_ids.numel()), dtype=torch.int32, device=device)
+    a_map = torch.zeros((local_topk_ids.numel()),
```

Comment:
nit: Can you add a comment about why (a_map + c2) need to be initialized to zeros instead of empty?